client/src/components/RelatedItemsAndOutfit.jsx

```diff
@@ -1,21 +1,50 @@
 import React from 'react';
 import Carousel from './Carousel.jsx';
+import { productsMock, productInfoMock, productStylesMock, relatedProductsMock } from '../mockData/productInfoMock.js';
+import { reviewsMock, reviewsMetaMock } from '../mockData/reviewsMock.js';
```

Comment:
Not passing data down from the top?

I thought that's how we were handling state, and this isn't something I'd hold up the PR over, but I figured if it were me, I'd have gone ahead and modeled passing the data down from ProductDetailPage.

Reply:
you're right, just didn't want to get in the way of other people's stuff. Keep my mess within my area until we can coordinate our big picture otp-level state

Reply:
But that is the better way to do it hahaha